templates/page.html

```diff
@@ -118,24 +118,19 @@ <h3>Free</h3>
         <h3>Structure your analysis</h3>
         <figure class="figure">
           <div class="command-line">
-            <div class="line">version: 0.2.0</div>
-            <div class="line">code:</div>
-            <div class="line tab">files:</div>
-            <div class="line tab">- code/mycode.py</div>
+            <div class="line">version: 0.3.0</div>
             <div class="line">inputs:</div>
             <div class="line tab">files:</div>
+            <div class="line tab">- code/mycode.py</div>
```

Comment:
What about additional indentation for items under `files:` (here and below) so that it would resemble `parameters:`